internal/services/purview/purview_account_resource.go

```diff
@@ -278,3 +303,29 @@ func flattenPurviewAccountIdentity(identity *purview.Identity) interface{} {
 		},
 	}
 }
+
+func flattenPurviewAccountManagedResources(managedResources *purview.AccountPropertiesManagedResources) interface{} {
+	if managedResources == nil {
+		return make([]interface{}, 0)
+	}
+
+	resourceGroup := ""
+	if managedResources.ResourceGroup != nil {
+		resourceGroup = *managedResources.ResourceGroup
+	}
+	storageAccount := ""
+	if managedResources.StorageAccount != nil {
+		storageAccount = *managedResources.StorageAccount
+	}
+	eventHubNamespace := ""
+	if managedResources.EventHubNamespace != nil {
+		eventHubNamespace = *managedResources.EventHubNamespace
+	}
+	return []interface{}{
+		map[string]interface{}{
+			"resource_group":      resourceGroup,
+			"storage_account":     storageAccount,
+			"event_hub_namespace": eventHubNamespace,
+		},
```

Comment:
since these are all ID's, can we suffix these with `_id`:

```suggestion
			"resource_group_id":      resourceGroup,
			"storage_account_id":     storageAccount,
			"event_hub_namespace_id": eventHubNamespace,
		},
```